dist/licenses.txt

```diff
@@ -0,0 +1,1622 @@
+@actions/core
+MIT
+The MIT License (MIT)
+
+Copyright 2019 GitHub
```

Comment:
Why Github?

Reply:
It is created by typescript GitHub action template. Changed.

Reply:
It is automatically created after packaging source code. It has been removed.